packages/process/src/node/raw-process.ts

```diff
@@ -69,10 +69,19 @@ export class RawProcess extends Process {
            normalize the error handling by calling the error handler
            instead.  */
         try {
-            this.process = spawn(
-                options.command,
-                options.args,
-                options.options);
+            if (options.command === 'node' && options.args) {
+                this.process = fork(
+                    options.args[0],
+                    [],
```

Comment:
Are you on purpose not passing arguments?

Reply:
yes, there aren't any args to be passed to the script when debugging. This is a quick hack as debug is the only place node is spawned AFAIK